tasks/main.yml

```diff
@@ -1,32 +1,55 @@
 ---
-- name: Ensure /root/.aws/ is there
+- name: Update user directory
+  set_fact:
+    aws_cwa_user_path: "/home/{{ aws_cwa_local_user }}"
+  when:
+    - aws_cwa_local_user != "root"
+    - aws_cwa_user_path == "/root" # When not default, assume user has overridden.
```

Comment:
Please use `and`, I certainly can never remember if an array is `and` or `or`

Reply:
Arrays are `and`. I think this is much clearer than long lines of `and` and `or` but I'll switch back if you'd like.